config.m4

```diff
@@ -46,101 +58,93 @@ if test "$PHP_MONGO" != "no"; then
     ;;
   esac
 
-fi
-
-PHP_ARG_ENABLE(coverage,  whether to include code coverage symbols,
-[  --enable-coverage         Mongo: Enable code coverage symbols, maintainers only!], no, no)
-
-if test "$PHP_COVERAGE" = "yes"; then
-
-  if test "$GCC" != "yes"; then
-    AC_MSG_ERROR([GCC is required for --enable-coverage])
-  fi
-  
-  dnl Check if ccache is being used
-  case `$php_shtool path $CC` in
-    *ccache*[)] gcc_ccache=yes;;
-    *[)] gcc_ccache=no;;
-  esac
+  if test "$PHP_COVERAGE" = "yes"; then
+    if test "$GCC" != "yes"; then
+      AC_MSG_ERROR([GCC is required for --enable-coverage])
+    fi
 
-  if test "$gcc_ccache" = "yes" && (test -z "$CCACHE_DISABLE" || test "$CCACHE_DISABLE" != "1"); then
-    AC_MSG_ERROR([ccache must be disabled when --enable-coverage option is used. You can disable ccache by setting environment variable CCACHE_DISABLE=1.])
-  fi
-  
-  lcov_version_list="1.5 1.6 1.7 1.9 1.10 1.11"
-
-  AC_CHECK_PROG(LCOV, lcov, lcov)
-  AC_CHECK_PROG(GENHTML, genhtml, genhtml)
-  PHP_SUBST(LCOV)
-  PHP_SUBST(GENHTML)
-
-  if test "$LCOV"; then
-    AC_CACHE_CHECK([for lcov version], php_cv_lcov_version, [
-      php_cv_lcov_version=invalid
-      lcov_version=`$LCOV -v 2>/dev/null | $SED -e 's/^.* //'` #'
-      for lcov_check_version in $lcov_version_list; do
-        if test "$lcov_version" = "$lcov_check_version"; then
-          php_cv_lcov_version="$lcov_check_version (ok)"
-        fi
-      done
-    ])
-  else
-    lcov_msg="To enable code coverage reporting you must have one of the following LCOV versions installed: $lcov_version_list"      
-    AC_MSG_ERROR([$lcov_msg])
-  fi
+    dnl Check if ccache is being used
+    case `$php_shtool path $CC` in
+      *ccache*[)] gcc_ccache=yes;;
+      *[)] gcc_ccache=no;;
+    esac
 
-  case $php_cv_lcov_version in
-    ""|invalid[)]
-      lcov_msg="You must have one of the following versions of LCOV: $lcov_version_list (found: $lcov_version)."
+    if test "$gcc_ccache" = "yes" && (test -z "$CCACHE_DISABLE" || test "$CCACHE_DISABLE" != "1"); then
```

Comment:
I don't see any actual changes in this block, or any of the follow blocks.
These is just ws?

Reply:
Not just whitespace. It moves the processing of `PHP_COVERAGE` and `PHP_MONGO_SASL` to only take place if `PHP_MONGO` is also configured.